crates/bevy_render/src/camera/projection.rs

```diff
@@ -197,14 +197,16 @@ pub enum ScalingMode {
 ///
 /// Note that the scale of the projection and the apparent size of objects are inversely proportional.
 /// As the size of the projection increases, the size of objects decreases.
+///
+/// Note also that the view frustum is centered at the origin.
 #[derive(Component, Debug, Clone, Reflect)]
 #[reflect(Component, Default)]
 pub struct OrthographicProjection {
     /// The distance of the near clipping plane in world units.
     ///
     /// Objects closer than this will not be rendered.
     ///
-    /// Defaults to `0.0`
+    /// Defaults to `-1000.0`
```

Comment:
```suggestion
    /// Defaults to `-1000.0`, for consistency with [`Camera2dBundle`].
```

Nitpick, not sure it's necessary.

Reply:
I think it adds more confusion than removes. I'd prefer if we keep it the way rdrpenguin04 wrote it.